gems/canvas_sanitize/lib/canvas_sanitize/canvas_sanitize.rb

```diff
@@ -90,6 +90,7 @@ def self.clean_style_value(config, value)
                    'role',
                    'lang',
                    'dir',
+                   :data,
```

Comment:
Would you mind adding a comment there indicating that?  It would make it easier for those looking in the future to understand the reasoning.